src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,21 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        final String errorMessage = "Can't get min value from list: " + numbers;
+
+        if (numbers.isEmpty()) {
+            throw new RuntimeException(errorMessage);
+        }
+
+        return Arrays.stream(numbers.stream()
+                        .collect(Collectors.joining(","))
+                        .split(","))
+                .distinct()
+                .map(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min(Integer::compareTo)
```

Comment:
```suggestion
                .min()
```
It’s  default behavior